retroshare-gui/src/gui/gxs/GxsIdTreeWidgetItem.cpp

```diff
@@ -201,7 +201,7 @@ void GxsIdTreeItemDelegate::paint(QPainter *painter, const QStyleOptionViewItem
     // additional Qt role only to determine the number of destinations.
 
     if(dt == "")
-        ownOption.icon = FilesDefs::getIconFromQtResourcePath(":/icons/svg/people2.svg");
+        ownOption.icon = FilesDefs::getIconFromQtResourcePath(":/icons/png/people2.png");
```

Comment:
Why this change? svg is most of the time much smaller than png and we already have memory problems.

Reply:
svg resize not works on a treeitem bug, but png works

Reply:
ins mails columns is this bug
![image](https://github.com/user-attachments/assets/713fee29-70cb-4ec9-a39b-379ef3c89136)

Reply:
as png version 5,23 kb
in svg 3.38kb